publisher/guides/api-usage.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: How to use the Rest API
+---
+
+import YouTube from 'react-youtube';
+
+# How to use the Rest API with authentification
+
+All changes to data using the API require the authentication of the user. In addition, only authenticated users can query non-public data. 
+An authorized user must have the right to modify posts in order to modify or query data. Here, the possibilities for authentication with the Wordpress 
+Application Password are presented. Furthermore, it is explained how the API can be tested with Postman and Application Password. 
```

Comment:
Als Vorschlag: "To authenticate the requests against the Podlove Publisher API we recommend the WordPress application password functionality. In the example you can also see how you can test the API with Postman and the WordPress application passwords."

Reply:
Dein Vorschlag ist als Einleitung gemeint, richtig?